`uint8arraylist` missing as peer dependency

Generated files contains the following line:

```
import type { Uint8ArrayList } from "uint8arraylist";
```

Depending on how npm manages the dependencies, that may lead to issues such as:

```
src/proto/store_v2beta4.ts:271:5 - error TS2322: Type 'import("/home/fryorcraken/src/status-im/js-waku/node_modules/protons-runtime/node_modules/uint8arraylist/dist/src/index").Uint8ArrayList' is not assignable to type 'import("/home/fryorcraken/src/status-im/js-waku/node_modules/uint8arraylist/dist/src/index").Uint8ArrayList'.

271     return encodeMessage(obj, WakuMessage.codec());
```

Proposal: declare the right `uint8arraylist` as a peer dependency of `protons-runtime`.

--- a/packages/protons-runtime/package.json
+++ b/packages/protons-runtime/package.json
@@ -151,6 +151,9 @@
     "protobufjs": "^7.0.0",
     "uint8arraylist": "^2.3.2"
   },
+  "peerDependencies": {
+    "uint8arraylist": "^2.3.2"
+  },
   "devDependencies": {
     "aegir": "^37.0.5"
   }

